### DISCUSSION TITLE
Is there any method to show the icon in git-revision plugin?

### DISCUSSION THREAD

**Discussion by mendax1234** [squidfunk/mkdocs-material#5996] [Q&A] [answered]
<img width="182" alt="image" src="https://github.com/squidfunk/mkdocs-material/assets/62934294/90f0a5d3-bad0-4175-ae1e-55935dffa1b4">
I find that there are icons shown on the mkdocs-material.

However, if l only use the git revision localized plugin. I can't see the icons shown.
<img width="221" alt="image" src="https://github.com/squidfunk/mkdocs-material/assets/62934294/22088cba-857c-4ec9-a2c6-a94c98aa1bdb">

Instead, only see something like this. Is that a normal case?

**Comment by squidfunk** [comment, marked as answer]
Thanks for asking! The [git revision functionality](https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#revisioning) was revamped and improved as part of [Insiders](https://squidfunk.github.io/mkdocs-material/insiders/), and is currently reserved to [sponsors](https://github.com/sponsors/squidfunk). Additionally, we added new functionality to show the [authors](https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#document-authors) or [contributors](https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#document-contributors).  If you sponsor the project and build your project with Insiders, you will get the new revised look. Once [the next funding goal is hit](https://squidfunk.github.io/mkdocs-material/insiders/#14000-goats-horn), we'll ship this as part of the community edition – as always. Your options: sponsor the project or wait ☺️
